scripts/after_prepare.js

```diff
@@ -55,6 +55,28 @@ var PLATFORM = {
   }
 };
 
+function updateStringsXml(contents) {
```

Comment:
I see this new function was added, but it isn't being called anywhere.  Am I missing something?

Reply:
copyKey(PLATFORM.ANDROID) should have the updateStringsXml callback, but i do not know where it went.